Dockerfile

```diff
@@ -0,0 +1,57 @@
+# This dockerfile is used to test containerd within a container
+#
+# usage: 
+# 1.) docker build -t containerd-test -f Dockerfile .
+# 2.) docker run -it --privileged containerd-test
+# 3.) make test
+#
+
+FROM golang:1.9.2
+
+RUN apt-get update && apt-get install -y \
+	autoconf \
+	automake \
+	btrfs-tools \
+	curl \
+	g++ \
+	git \
+	libapparmor-dev \
+	libseccomp-dev \
+	libtool \
+	tar \
+	unzip
+
+# Install proto3
+RUN set -x \
+	&& mkdir -p /go/src/github.com/google \
+	&& cd /go/src/github.com/google \
+	&& git clone "https://github.com/google/protobuf.git" \
+	&& cd protobuf \
+	&& ./autogen.sh \
+	&& ./configure \
+	&& make \
+	&& make check \
+	&& make install \
+	&& ldconfig
+
+# Install runc
+RUN set -x \
+	&& mkdir -p /go/src/github.com/opencontainers \
+	&& cd /go/src/github.com/opencontainers \
+	&& git clone "https://github.com/opencontainers/runc.git" \
```

Comment:
I think this should use the pinned version from `RUNC.md` since that's what we document as being supported.